deployment/stacks/ui.ts

```diff
@@ -0,0 +1,239 @@
+import { Construct } from "constructs";
+import {
+  aws_cloudfront as cloudfront,
+  aws_cloudfront_origins as cloudfrontOrigins,
+  aws_iam as iam,
+  aws_s3 as s3,
+  Duration,
+  RemovalPolicy,
+  aws_certificatemanager as acm,
+} from "aws-cdk-lib";
+import { WafConstruct } from "../constructs/waf";
+import { addIamPropertiesToBucketAutoDeleteRole } from "../utils/s3";
+import { IManagedPolicy } from "aws-cdk-lib/aws-iam";
+import { isLocalStack } from "../local/util";
+
+interface CreateUiComponentsProps {
+  scope: Construct;
+  stage: string;
+  project: string;
+  isDev: boolean;
+  iamPermissionsBoundary: IManagedPolicy;
+  iamPath: string;
+  cloudfrontCertificateArn?: string;
+  cloudfrontDomainName?: string;
+  vpnIpSetArn?: string;
+  vpnIpv6SetArn?: string;
+}
+
+export function createUiComponents(props: CreateUiComponentsProps) {
+  const {
+    scope,
+    stage,
+    project,
+    isDev,
+    iamPermissionsBoundary,
+    iamPath,
+    cloudfrontCertificateArn,
+    cloudfrontDomainName,
+    /*
+     * vpnIpSetArn,
+     * vpnIpv6SetArn,
+     */
+  } = props;
+
+  // S3 Bucket for UI hosting
+  const uiBucket = new s3.Bucket(scope, "uiBucket", {
+    encryption: s3.BucketEncryption.S3_MANAGED,
+    removalPolicy: RemovalPolicy.DESTROY,
+    autoDeleteObjects: true,
+    enforceSSL: true,
+  });
```

Comment:
```suggestion
  const loggingBucket = s3.Bucket.fromBucketName(
    this,
    "LoggingBucket",
    `cms-cloud-${Aws.ACCOUNT_ID}-${Aws.REGION}`
  );
    
  const uiBucket = new s3.Bucket(scope, "uiBucket", {
    encryption: s3.BucketEncryption.S3_MANAGED,
    removalPolicy: RemovalPolicy.DESTROY,
    autoDeleteObjects: true,
    enforceSSL: true,
    blockPublicAccess: s3.BlockPublicAccess.BLOCK_ALL,
    versioned: true,
    serverAccessLogsBucket: loggingBucket,
    serverAccessLogsPrefix: `AWSLogs/${Aws.ACCOUNT_ID}/s3/`,
  });
```

Reply:
Per slack discussion, tabling this for now.